sdk/avalon_sdk/connector/direct/jrpc/jrpc_worker_registry.py

```diff
@@ -179,13 +201,26 @@ def worker_register(self, worker_id, worker_type, org_id,
         Returns:
         JRPC response with worker registry status.
         """
+
+        if worker_type is not None:
+            if isinstance(worker_type, WorkerType):
+                worker_value = worker_type.value
+            elif isinstance(worker_type, int) and \
+                    worker_type in range(1, 3):
```

Comment:
Can we use enum for workerType. Currently only "TEE-SGX" worker type is supported

Reply:
done